kuma/static/styles/minimalist/components/_search-widget.scss

```diff
@@ -1,26 +1,38 @@
+@import '../utils/utility-mixins.scss';
+
 .header-search {
-    grid-area: S;
-    display: flex;
-    flex: 2 1;
-    flex-direction: row;
-    align-items: center;
+    $mobile-search-height: 68px;
+    $icon-size: 24px;
     box-sizing: border-box;
-    padding: 2px 8px;
-    border: 2px solid $text-color;
-    border-radius: 20px;
-    height: 40px;
-    min-width: 80px;
-
-    .search-icon {
-        flex: 0 0 25px;
-        fill: $text-color;
+    position: absolute;
+    top: -$mobile-search-height;
+    right: 0;
+    width: 100vw;
+    height: $mobile-search-height;
+
+    form {
+        background: $neutral-100;
+        grid-area: S;
+        display: flex;
+        flex: 2 1;
```

Comment:
The above rule and the rule `grid-area: S` has no effect as the form is not a grid item. We can safely remove it. `grid-area: S` should be on `.header-search` instead. Our grid and general layout is... a little scattered at the moment :smile\_cat:

Reply:
Removed `flex: 2 1;` and moved `grid-area: S` to `.header-search`. Thank you!